src/components/SearchForm/SearchForm.js

```
@@ -39,7 +39,7 @@ const SearchForm = () => {
       await querySearchTerms(formattedTerm)
       await setLoading(false)
     } catch(error){
-      console.log(error)
+      setError(true)
```

You're swallowing the error here and setting a flag, I'm guessing to show some kind of UI. How would you as a developer know that your user encountered this error and debug it later?